rholang/src/main/scala/coop/rchain/rholang/interpreter/Reduce.scala

```diff
@@ -52,7 +52,7 @@ trait Reduce[M[_]] {
 
 class DebruijnInterpreter[M[_], F[_]](
     tuplespaceAlg: Tuplespace[M],
-    private val urnMap: Map[String, Par]
+    urnMap: Map[String, Par]
```

Comment:
What's the reason for relaxing the access constraints to `urnMap`?

Reply:
The `private val` modifier is redundant because constructor arguments aren't accessible in class instances without a `val` modifier.